From c40b44bee4050ea0aa0969a3fd3811c5263eca55 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Thu, 2 Jan 2020 10:34:12 +0800
Subject: [PATCH] dt-bindings: display: rockchip: Add  RGB documentation for
 rv1126

There is a RGB output port on rv1126, add documentation for it.

Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
Change-Id: I012cafc394b3056b5deba46350d99635b7dc9b5a
---
 .../devicetree/bindings/display/rockchip/rockchip-rgb.txt        | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/display/rockchip/rockchip-rgb.txt b/Documentation/devicetree/bindings/display/rockchip/rockchip-rgb.txt
index fbd7660cded6..df46a1561a73 100644
--- a/Documentation/devicetree/bindings/display/rockchip/rockchip-rgb.txt
+++ b/Documentation/devicetree/bindings/display/rockchip/rockchip-rgb.txt
@@ -11,6 +11,7 @@ Required properties:
 	- "rockchip,rk3308-rgb";
 	- "rockchip,rk3368-rgb";
 	- "rockchip,rv1108-rgb";
+	- "rockchip,rv1126-rgb";
 
 Optional properties:
 - phys : phandle for the PHY device
-- 
2.35.3

